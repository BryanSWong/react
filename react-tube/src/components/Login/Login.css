.Button {
    width: 100%;
    padding: 12px 20px;
    margin: 8px 0;
    box-sizing: border-box;

}

@media (min-width: 500px) and (min-height: 400px){
    .Button{
        width: 350px;
        height: 300px;
    }
}

@media (min-width: 500px) and (min-height: 401px){
    .Button{
        width: 450px;
        height: 400px;
    }
}

@media (min-width: 1000px) and (min-height: 700px){
    .Button{
        width: 700px;
        height: 600px;
    }
}