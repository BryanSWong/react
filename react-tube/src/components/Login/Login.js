import React, {Component} from 'react';

import classes from './Login.css';

class Login extends Component {

  render() {
    return (
      <div>
        <h3 id="formTitle" className={classes.Title}>Sign In</h3>
        <form className={classes.SignInForm}>
          <input className={classes.FormInput} type="text" id="username"
                 placeholder="Enter your username" />
          <input className={classes.FormInput} type="password" id="password"
                 placeholder="Enter your password"/>
          <input className={classes.LogInButton} type="submit" id="submit" value="Log In"/>
        </form>
      </div>
    );
  }

}

export default Login;