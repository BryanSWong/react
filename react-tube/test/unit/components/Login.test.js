import React from 'react';

import Login from '../../../src/components/Login/Login';

describe('Login Component', () => {

    const component = shallow(<Login debug />);

    it('should render correctly in "debug" mode', () => {
        expect(component).toMatchSnapshot();
    });

});