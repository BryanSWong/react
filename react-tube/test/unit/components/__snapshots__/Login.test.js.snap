// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component should render correctly in "debug" mode 1`] = `
<div>
  <h3
    id="formTitle"
  >
    Sign In
  </h3>
  <form>
    <input
      id="username"
      placeholder="Enter your username"
      type="text"
    />
    <input
      id="password"
      placeholder="Enter your password"
      type="password"
    />
    <input
      id="submit"
      type="submit"
      value="Log In"
    />
  </form>
</div>
`;
